src/Leetcode1004MaxConsecutiveOnesIII.java

```diff
@@ -0,0 +1,35 @@
+
+public class Leetcode1004MaxConsecutiveOnesIII {
+    public static int longestOnes(int[] nums, int k) 
+    {
+        int j = 0;
+        int i = 0;
+
+        while (i < nums.length) 
```

Comment:
shouldn't you use a for loop instead of a while loop in this situation?

Reply:
fixed